package/gluon-respondd/files/etc/hotplug.d/iface/10-gluon-respondd

```diff
@@ -6,29 +6,17 @@
 DEVLIST=/var/run/gluon-respondd.devs
 
 ifname_to_dev () {
-	json_load "$(ubus call network.interface.$1 status)"
-	json_get_var dev device
+  json_load "$(ubus call network.interface.$1 status)"
+  json_get_var dev device
 
-	echo "$dev"
+  echo "$dev"
 }
 
 case "$ACTION" in
-	ifdown)
-		sed "/ $INTERFACE$/d" $DEVLIST > $DEVLIST.new
-		mv $DEVLIST.new $DEVLIST
-		;;
-	ifup)
-		DEVICE="$(ifname_to_dev "$INTERFACE")"
-		MESH="$(cat "/sys/class/net/$DEVICE/batman_adv/mesh_iface" 2>/dev/null)"
+  ifup)   
+    DEVICE="$(ifname_to_dev "$INTERFACE")"
 
-		[ "$MESH" = "bat0" -o "$INTERFACE" = "client" ] || exit 0
-
-		DEVS=$(cat $DEVLIST 2>/dev/null; echo $DEVICE $INTERFACE)
-
-		echo "$DEVS" | sort -u > $DEVLIST.new
-		mv $DEVLIST.new $DEVLIST
-
-		/etc/init.d/gluon-respondd restart_if_running &
-
-		;;
+    [ "$DEVICE" == $(cat /lib/gluon/respondd/client.dev) ] && 
```

Comment:
`==` is invalid here, it should be `=`. Even better would be to change it into `!=` and `&&` into `||`, so the expression always returns true, otherwise the whole script might return false.